### DISCUSSION TITLE
FileZilla not available anymore

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#159] [General]
@KnifMelti Have you seen that ?
https://github.com/microsoft/winget-pkgs/issues/78578#issuecomment-1238126217

Maybe other apps will follow for same reason ? 🤔

**Comment by KnifMelti** [comment]
Yup.. ..**Scoop** solved it with "../_Our package now uses https://github.com/ScoopInstaller/Binary as the file source_./.."
https://github.com/ScoopInstaller/Binary/tree/master/filezilla - how they now plan to manually/automatically update this repo?

**Microsoft** maybe need to reach out to make an exception "_../Our infrastructure is meant to be used only by humans and FileZilla itself, not third-party installers./.._"

This is why whitelist/constant monitoring is of importance...
